build-packages/magento-scripts/lib/commands/start.js

```diff
@@ -111,9 +114,15 @@ module.exports = (yargs) => {
             logger.logN(`Magento Admin panel credentials: ${logger.style.misc(magentoConfiguration.user)} - ${logger.style.misc(magentoConfiguration.password)}`);
             logger.note(`MySQL credentials, containers status and project information available in ${logger.style.code('npm run status')} command.`);
             logger.log('');
+
+            if (analytics && process.isFirstStart) {
+                await googleAnalytics.trackTiming('CMA first development time', Date.now() / 1000 - timeStamp);
```

Comment:
Should there be a try/catch in case of error?

Reply:
Also, maybe we can not only track first start time, but also a usual start-up time.
And probably we'll need to add more data to this type of data, like platform that our code is run on and system configuration (cpu model, ram amount) to get some idea about how system configuration affects general CMA performance.